build.sh.tpl

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+AWS="/usr/local/bin/aws"
+AWS_DEFAULT_REGION=$AWS_DEFAULT_REGION
+RC_DIR=$RC_DIR
+RC_VERSION=$version
+S3_BUCKET=$S3_BUCKET
```

Comment:
What's the point of these? It seems like you wanted env vars to get assigned even if they are not, but I can't think of a reason to do it.
I think that even if you add `set -u` at the top it will still not fail here, if not defined, or will it?

Reply:
These variables are used here for substitution, see jenkins job config for details